conf/drupal/config/entity_usage.settings.yml

```diff
@@ -1,9 +1,7 @@
 track_enabled_base_fields: true
 local_task_enabled_entity_types:
-  - media
-  - node
-  - taxonomy_term
-usage_controller_items_per_page: 25
+  - view
+usage_controller_items_per_page: 1
```

Comment:
Did you mean to change to 1 per page?

Reply:
No I didn't. Definitely didn't set that either.